docs/insomnia/introduction-to-plugins.md

```diff
@@ -91,23 +91,55 @@ Example: Plugin package.json file
 }
 ```
 
+### Expose Response Body Text
+
+When developing a new plugin, you may want to expose content in the response body. In the following example, we demonstrate how you may prompt the user to enter content into a prompt and subsequently show this content when the user sends the request. 
+
+```
+const bufferToJsonObj = buf => JSON.parse(buf.toString('utf-8'));
+const jsonObjToBuffer = obj => Buffer.from(JSON.stringify(obj), 'utf-8');
+
+module.exports.responseHooks = [
+  async ctx => {
+    try{
+      const resp = bufferToJsonObj(ctx.response.getBody());
+      
+      // Modify
+      resp.__randomNumber = Math.random();
+
+      // If you want something from a user, use a prompt:
+      const promptResult = await ctx.app.prompt('Type something', { defaultValue: 'abcd' });
+      resp.__customValue = promptResult;
+
+      ctx.response.setBody(jsonObjToBuffer(resp));
+    } catch {
+      // no-op
+    }
+  }
+]
+```
+_This example specifically deals with making timestamps human readable. Update the functionality as needed._
+
 ## Manage Plugins
-Plugins can be added from the Plugins tab within the application preferences dialog. They can also be downloaded and installed directly from npm.
+
+Plugins can be added from the **Plugins** tab in the **Preferences** menu. They can also be downloaded and installed directly from [NPM](https://insomnia.rest/plugins).
```

Comment:
We can also enable and disable plugins within the app now! The little toggle switch next to a plugin